src/harness/sas.py

```diff
@@ -304,9 +304,124 @@ def InjectCpiUser(self, request):
     _RequestPost('https://%s/admin/injectdata/cpi_user' % self._base_url,
                  request, self._tls_config)
 
+  def TriggerCreateActivityDump(self, request):
+    _RequestPost('https://%s/admin/trigger/create_full_activity_dump' % self._base_url,
+                 request, self._tls_config)
+
+  def InjectPeerSas(self,request):
+    _RequestPost('https://%s/admin/injectdata/peer_sas' % self._base_url,
+                 request, self._tls_config)
+
   def _GetDefaultAdminSSLCertPath(self):
     return os.path.join('certs', 'admin_client.cert')
 
   def _GetDefaultAdminSSLKeyPath(self):
     return os.path.join('certs', 'admin_client.key')
 
+def GetServer():
+  config_parser = ConfigParser.RawConfigParser()
+  config_parser.read(['sas.cfg'])
+  server_base_url = config_parser.get('HttpServer', 'BaseUrl')
+  server_port = int(config_parser.get('HttpServer', 'Port'))
+  return HttpServer({'baseUrl': server_base_url, 'port': server_port})
+
+class HttpServer(sas_interface.HttpServerInterface):
+  """Test Harness acting as a Http Server to receive Pull/GET and Push/POST
+   Requests from SAS Under Test
+  """
+  def __init__(self, server_details):
```

Comment:
Please pass args in separately rather an as a dict -- I think it's more in keeping with normal Python style and it's self-documenting.